tests/app/data_model/test_session_store.py

```diff
@@ -133,13 +131,13 @@ def test_session_store_stores_trading_as_value_if_present(
 
         session_store = SessionStore("user_ik", "pepper", "eq_session_id")
 
-        assert hasattr(session_store.session_data, "trad_as") is True
+        assert hasattr(session_store.session_data, "language_code") is True
```

Comment:
Existing but check the value?. This would be true even if value didn't exist as it defaults to None.

Reply:
I've reworded them so it makes sense now, this test and the one below.

Reply:
My point is that the test on its own isnt very useful without checking the value. It will have the attr because it always does, but it doesn't check the attr get set properly.

Reply:
Ok, now it tests if language code is being created from `session_data` and added as `session_store.session_data` instead of just the existence of it. Thanks!